sdk/highlight-run/src/plugins/observe.ts

```diff
@@ -115,30 +125,25 @@ export class Observe extends Plugin<ObserveOptions> implements LDPlugin {
 
 					const eventAttributes: Attributes = {
 						[FEATURE_FLAG_KEY_ATTR]: hookContext.flagKey,
-						[FEATURE_FLAG_VARIANT_ATTR]: JSON.stringify(
-							detail.value,
-						),
+						[FEATURE_FLAG_VALUE_ATTR]: JSON.stringify(detail.value),
+						[FEATURE_FLAG_IN_EXPERIMENT_ATTR]:
+							detail.reason?.inExperiment,
```

Comment:
Will this send `false` if `inExperiment` is false (or `reason` is falsey)? If so we might want to update the tech spec, which currently says this attribute will be omitted unless it would be `true`.

(I think it's fine to send `false`s - the "omit" behavior is what the LD SDK currently does mostly because of the default behavior of the Go `encoding/json` library, which is not a great reason.)

Reply:
I think omitting makes sense for consistence with the tech spec and existing implementations. I'll update this, thanks.

Reply:
FWIW the spec already does say only to include it when it is true.

Reply:
Yeah, I was suggesting to change the spec, but changing the code is fine too.

Reply:
So, the SDK only includes it in the reason when it is true, so likely this would actually work. It would be undefined and if it uses standard JSON serialization it will be dropped.

Reply:
For clarity, I'm making this only include the values when truthy (with a comment explaining this behavior).